snapshot-labs/snapshot
# fix: wallectconnect with op wallet

To support https://www.candidewallet.com/ wallet using walletconnect and wallets that support multiple chains

### Summary
- Make chains = [] to support all networks while connecting using Wallet-connect https://github.com/snapshot-labs/snapshot/pull/4700#discussion_r1584993894
- Make `eth_signTypedData_v4` optional https://github.com/snapshot-labs/snapshot/pull/4700#discussion_r1587396277
- Safe wallet should work with all networks, previously only mainnet was working using wallet connect

### How to test

1. Go to https://snapshot-git-fix-voting-with-op-walletconnect-snapshot.vercel.app/
2. Connect wallet with wallet connect and everything should work as before
